core/src/main/java/org/elasticsearch/monitor/jvm/JvmInfo.java

```diff
@@ -307,7 +314,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.field(Fields.GC_COLLECTORS, gcCollectors);
         builder.field(Fields.MEMORY_POOLS, memoryPools);
 
-        builder.field(Fields.USING_COMPRESSED_OOPS, usingCompressedOops == null ? "unknown" : Boolean.toString(usingCompressedOops));
+        builder.field(Fields.USING_COMPRESSED_OOPS, useCompressedOops);
```

Comment:
@nik9000 Yes, that's exactly how it will be on the backport but it's not needed here.